sklearn/tests/test_public_functions.py

```diff
@@ -122,6 +122,7 @@ def _check_function_param_validation(
     "sklearn.model_selection.train_test_split",
     "sklearn.random_projection.johnson_lindenstrauss_min_dim",
     "sklearn.svm.l1_min_c",
+    "cluster.cluster_optics_xi",
```

Comment:
As the previous lines, you need a fully qualified path.

```suggestion
    "sklearn.cluster.cluster_optics_xi",
```

Please put this line in the correct alphabetic order.